package.json

```diff
@@ -4,13 +4,10 @@
   "description": "",
   "main": "dist/generateBaseFieldsInserts.js",
   "scripts": {
-    "build": "npm run cleanup && npm run build:node",
-    "build:node": "tsc -p tsconfig.json",
-    "cleanup": "rimraf dist",
     "lint": "eslint ./src --ext .ts",
     "test": "echo \"no tests\" && exit 1",
-    "generateBaseFieldsInserts": "node dist/generateBaseFieldsInserts.js",
-    "generateApplicationFormJson": "node dist/generateApplicationFormJson.js"
+    "generateBaseFieldsInserts": "ts-node src/generateBaseFieldsInserts.ts",
```

Comment:
Sanity check: do we need to specify the file extensions?

Reply:
Oh! Excellent catch, we do not. That never even occurred to me!